Example/Pods/Local Podspecs/Swiftilities.podspec.json

```diff
@@ -38,6 +38,10 @@
       ]
     },
     {
+      "name": "Collections",
```

Comment:
#req pretty sure Collections has a dependency on Math. That dependency should be explicit.